### PR TITLE
RHDEVDOCS-4482: Added the feature creating a Kafka event sink

### PR DESCRIPTION
**Pull request by shipsing**
Purpose: To resolve https://issues.redhat.com/browse/RHDEVDOCS-4482
Version(s): enterprise-4.12 only

Link to docs preview: https://51377--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-12-release-notes.html#ocp-4-12-web-console

SME review:@invincibleJai 
QE review: @cardil 
Peer review: @gabriel-rh

### CODE REVIEW

**Comment by shipsing** [comment]
@pierDipi, Could you please help with SME review of this PR? Thanks

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -59,6 +59,11 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-web-console"]
 === Web console
 
+[id="ocp-4-12-developer-perspective"]
+==== Developer Perspective
+
+* With this update, in the *+Add* view of the *Developer* perspective, you can create a Kafka event sink to receive events from a particular source and send them to a Kafka topic.
```
Comment (invincibleJai) [inline release_notes/ocp-4-12-release-notes.adoc:65]: do we need to mention the prerequisite like user needs to install serverless operator and create knativeEventin and knativeKafka with sink enabeled and in dev console it's available in Add -> Event Sink -> kakfaSink will be shown in catolog

This doc talks about enabling it https://docs.openshift.com/container-platform/4.11/serverless/admin_guide/serverless-kafka-admin.html

flow can be seen here from our sprint demo https://drive.google.com/file/d/1WPuzLnz3DOuGv2uhIaVitALdlULrLaIt/view?usp=sharing

Comment (shipsing) [reply release_notes/ocp-4-12-release-notes.adoc]: @invincibleJai, This text is for Release Notes and we need not to dicuss about this event sink feature in details as we have already discussed about the prerequistes and procedure in detail under this PR: https://github.com/openshift/openshift-docs/pull/50536. The main intent is just to inform our readers about this new functionality. So, i removed this part` in the *+Add* view of the *Developer* perspective` to make the statement generic.

**Comment by invincibleJai** [comment]
LGTM

**Comment by shipsing** [comment]
@mgencur, Could you please help with QE review of this PR? Thanks

**Review by cardil — APPROVED** [review]
/lgtm

from QE

**Review by gabriel-rh — APPROVED** [review]
/lgtm

Looks good!

**Pull request merged**